packages/link-bins/src/index.ts

```diff
@@ -81,7 +81,7 @@ async function linkBins (
     warn: (msg: string) => void,
   },
 ) {
-  if (!allCmds.length) return
+  if (!allCmds.length) return ['']
```

Comment:
If you did this just to surpess a typescript error, then the right way is `[] as string[]`